Bottom: e42cef28f32cc67fd3f46030fda9b541dcc6ad96
Top:    b47a30d8e24a50ce4b38c71b3a560a2b66a46183
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 13:52:50 +0000

Refresh of fix-inverts

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index cbeea35..8e37f26 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -724,7 +724,8 @@ object3d_from_board_outline (void)
                                     -HACK_BOARD_THICKNESS / 2, /* Top */
 #endif
                                     board_appearance,
-                                    top_bot_appearance);
+                                    top_bot_appearance,
+                                    false); /* Don't invert */
 
   destroy_appearance (board_appearance);
   destroy_appearance (top_bot_appearance);
@@ -988,7 +989,8 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
                                     (side == TOP_SIDE) ? -HACK_BOARD_THICKNESS / 2 - HACK_COPPER_THICKNESS - HACK_MASK_THICKNESS : HACK_BOARD_THICKNESS / 2 + HACK_COPPER_THICKNESS, /* Top */
 #endif
                                     mask_appearance,
-                                    NULL);
+                                    NULL,
+                                    false); /* Don't invert */
 
   destroy_appearance (mask_appearance);
